core/requests/types.go

```diff
@@ -16,6 +16,22 @@ type SignMessage struct {
 	GuardianAddr string `json:"guardian"`
 }
 
+// SetSecurityModeNoExpireMessage is the JSON request the service is receiving
+// when a user wants to set the security mode
+type SetSecurityModeNoExpireMessage struct {
```

Comment:
SetSecurityModeNoExpireMessage is duplicated with UnsetSecurityModeNoExpireMessage

can rename it to SecurityModeNoExpire and reuse it

Reply:
updated